dot/rpc/http.go

```diff
@@ -138,15 +138,13 @@ func (h *HTTPServer) Start() error {
 			h.logger.Error("http error", "err", err)
 		}
 	}()
-
-	if !h.serverConfig.WSEnabled {
-		return nil
-	}
+	// TODO: Confirm if RPC & WS always enabled is correct
 
 	h.logger.Info("Starting WebSocket Server...", "host", h.serverConfig.Host, "port", h.serverConfig.WSPort)
 	ws := mux.NewRouter()
 	ws.Handle("/", h)
 	go func() {
+		// TODO: Confirm if this only accepts localhost
```

Comment:
@noot

Reply:
yeah don't really know if this is testable with unit tests... if you replaced 127.0.0.1 with your external IP it would probably resolve to localhost anyways right?

Reply:
@noot I think I'd probably need to decouple the validation logic and then run tests against that rather than the logic that branches from it